doc/mf6io/mf6ivar/dfn/prt-prp.dfn

```diff
@@ -189,7 +189,7 @@ in_record true
 tagged false
 repeating true
 longname release times
-description times to release, relative to the beginning of the simulation.
+description times to release, relative to the beginning of the simulation.  RELEASE_TIMES and RELEASE_TIMESFILE are mutually exclusive.  Explicit release times and release settings configured in the period block are additive and can be mixed and matched.
```

Comment:
fyi, think you need to escape underscores with a preceding backslash

Reply:
thanks @langevin-usgs